### PR TITLE
feat(alerts): Migration to backfill existing orgs with `alerts_member_write: False`

### PR DESCRIPTION
**Pull request by davidenwang**
This migration is required to run before https://github.com/getsentry/sentry/pull/22924 can be merged.

The migration simply backfills existing orgs with the `OrganizationOption` `alerts_member_write: False`. This will make sure that when we add the option from https://github.com/getsentry/sentry/pull/22924, members of existing orgs won't be able to edit alerts unless the org opts into that specifically.

### CODE REVIEW

**Comment by davidenwang** [comment]
may change things to make this PR not dependent on the migration, don't review for now

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1366,8 +1367,8 @@ def create_partitioned_queues(name):
                 "team:write",
                 "team:admin",
                 "org:integrations",
-                "alerts:write",
                 "alerts:read",
+                "alerts:write",
```
Comment (davidenwang) [inline src/sentry/conf/server.py:1371]: just fixing the ordering of `alerts:write` over here

File: src/sentry/migrations/0146_backfill_members_alert_write.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def backfill_existing_orgs(apps, schema_editor):
+    """
+    Backfill the OrganizationOption alerts_member_write to be False for existing orgs
+    """
+    Organization = apps.get_model("sentry", "Organization")
+    OrganizationOption = apps.get_model("sentry", "OrganizationOption")
+
+    for org in RangeQuerySetWrapperWithProgressBar(Organization.objects.filter(status=0)):
```
Comment (mattrobenolt) [inline src/sentry/migrations/0146_backfill_members_alert_write.py:21]: There's no index on `status=0`, so I'd suggest just iterating over `Organization.objects.all()` instead, and add a:

```python
if org.status != 0:
    continue
```

Comment (mattrobenolt) [reply src/sentry/migrations/0146_backfill_members_alert_write.py:21]: Yeah, just to add to this:

```
getsentry=# select count(*), status from sentry_organization group by status order by 1 desc;
 count  | status
--------+--------
 412569 |      0
   1317 |      2
    514 |      1
(3 rows)
```

By far, most of them are 0, so this filter will just be more efficient to do in Python and easier on our database.

**Review by mattrobenolt — COMMENTED** [review]
The migration is fine, but I dunno what else is going on here.

**Comment by davidenwang** [comment]
> The migration is fine, but I dunno what else is going on here.

tyty I'm actually gonna go ahead and split out the rest of these files into a separate PR as they are blocked on the migration anyways

File: src/sentry/migrations/0146_backfill_members_alert_write.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+# Generated by Django 1.11.28 on 2020-07-28 16:24
+from __future__ import unicode_literals
+
+from django.db import migrations, transaction
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+import logging
+
+ORGANIZATION_OPTION_KEY = 'sentry:alerts_member_write'
```
Comment (wedamija) [inline src/sentry/migrations/0146_backfill_members_alert_write.py:11]: Nit: Probably no need for this to be a constant.

**Review by wedamija — APPROVED** [review]
lgtm

**Pull request merged**